### DISCUSSION TITLE
get_miss_tps 作用是什么 如果很高会怎么样

### DISCUSSION THREAD

**Discussion by wz2cool** [apache/rocketmq#8122] [General]
![image](https://github.com/apache/rocketmq/assets/11556152/295fd204-2f73-42bf-a3e9-282a667770b5)
如图 感觉很高 不知道会出现什么问题

**Comment by cserwen** [comment]
说明拉取请求没有拉到消息，最可能的原因就是请求位点越界了（上界或下界都有可能），但是服务端会返回给客户端有效的位点，看起来是客户端存在问题，没有使用服务端返回的位点去拉取消息
